spec/large_object_store_spec.rb

```diff
@@ -65,7 +33,7 @@ def delete(key)
   it "cannot read corrupted objects" do
     store.write("a", ["a"*10_000_000]).should == true
     store.store.write("a_4", nil)
-    store.read("a").should == nil
+    store.read("a").class.should == NilClass
```

Comment:
it won't print 10 million characters if it fails :D

On Tue, Nov 17, 2015 at 9:58 AM, Ana Martinez <notifications@github.com>
wrote:

> In spec/large_object_store_spec.rb
> <https://github.com/anamartinez/large_object_store/pull/8#discussion_r45094334>
> :
>
> > @@ -65,7 +33,7 @@ def delete(key)
> >    it "cannot read corrupted objects" do
> >      store.write("a", ["a"*10_000_000]).should == true
> >      store.store.write("a_4", nil)
> > -    store.read("a").should == nil
> > +    store.read("a").class.should == NilClass
>
> What's the difference?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/anamartinez/large_object_store/pull/8/files#r45094334>
> .
>